tests/integration_tests/vectorstores/test_opensearch.py

```diff
@@ -23,6 +23,18 @@ def test_opensearch() -> None:
     assert output == [Document(page_content="foo")]
 
 
+def test_similarity_search_with_score() -> None:
+    """Test similarity search with score using Approximate Search."""
+    docsearch = OpenSearchVectorSearch.from_texts(
+        texts, FakeEmbeddings(), opensearch_url=DEFAULT_OPENSEARCH_URL
```

Comment:
could we add some metadata just to make sure it's being correctly returned

Reply:
Updated the test to include metadata